release/README.md

```diff
@@ -26,6 +26,15 @@ https://semver.org/
 
     *When the script is done a [GitHub actions workflow pipeline](/.github/workflows/release.yml) should've created a GitHub release from that tag.*
 
+1. Update the Welcoming Dashboards "What's New" section
+
+    Write about some new feature or change that is relevant for the user, e.g. for `v0.25` "- As an admin user, you can now create namespaces yourself using HNC ..."
+
+    Also remove the items in this section from two+ older minor versions, meaning if you release apps `v0.28` you can keep previous items that were added to the list in `v0.27` but remove the stuff that are from `v0.26`.
+
+    - Edit the [Grafana dashboard](../helmfile/charts/grafana-ops/files/welcome.md)
+    - Edit the [Opensearch dashboard](../helmfile/charts/opensearch/configurer/files/dashboards-resources/welcome.md)
+
```

Comment:
I think this should be done directly after feature freeze, else you'd update this after you've created the release.

Reply:
Right, very good point.